licenses/MOMENT_MIT.TXT

```diff
@@ -20,3 +20,4 @@ HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
 WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE.
+INGS IN THE SOFTWARE.
```

Comment:
@davidkirwan unfortunately the license-reporter tool is not perfect sometimes. This line can be removed as it shouldn't be here.